appsettings.json

```diff
@@ -17,5 +17,17 @@
   "AllowedHosts": "*",
   "ConnectionStrings": {
     "DefaultConnection": "server=localhost;port=3306;database=personaltool;user=personaltool;password=personaltool;"
+  },
+  "RandomTestSeedConfig": {
+    "Enabled": false,
+    "ClearExistingData": false,
+    "Seed": 123,
```

Comment:
setting seed to null should lead to a random seed

Reply:
A null seed leads to a random seed, that's already implemented, or do you mean it should be the default? In that case, I agree